web-console/knikubevirt/migrate-vm/migrate-vm.md

```diff
@@ -0,0 +1,19 @@
+# Migrate VM
+
+Clicking the action button next to any Virtual Machine in the List will present the user with the option to migrate the VM to another host.
+
+![Selecting a VM](img/1-0-list.png)
+
+A modal will be presented where the user can choose which Node the VM should be migrated to. 
+
+![Modal view](img/1-1-modal-info.png)
+
+![Modal migrate](img/1-2-modal-migrate.png)
+
+After the migration is kicked off, the user will see a toast notification letting them know it has started. The state for the VM will also be reflected in the list view.
```

Comment:
I'm not sure we have Toasts yet, but it's a nice thing to keep for the future. @beanh66 - would you suggest we pull this out for now? Or is it okay to dream? :)

Reply:
@lizsurette Correct we do not have toasts in the console anymore, but we will be looking to add them back in. Since this is a new component to the react console, I think we should be using PF4 toasts instead of PF3.